lib/limits.c

```diff
@@ -119,7 +119,8 @@ set_umask(const char *value)
 /* Counts the number of user logins and check against the limit */
 static int check_logins (const char *name, const char *maxlogins)
 {
-	unsigned long limit, count;
+	unsigned long limit;
+	int exceeded = -1;
```

Comment:
Please, don't initialize.  Prefer assignment closer to use.  Initialization of variables can hide logic bugs.